html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <div>Email</div>
+    <label for="email">
+      <input type="text">
```

Comment:
inputs should always have a `name` attribute, the `name` attribute will become important down the road. The `name` attribute should be set to a value that matches the data the input is collecting. So for this input setting `name` to `email` would be ideal.